modules/analysis-common/src/test/resources/rest-api-spec/test/analysis-common/40_token_filters.yml

```diff
@@ -568,3 +568,98 @@
             filter:    [my_truncate]
     - length: { tokens: 1 }
     - match:  { tokens.0.token: foo }
+
+---
+"pattern_capture":
```

Comment:
Just curious: are these new test because I cannot find a corresponding file where they might be moved from?